src/onegov/org/forms/event.py

```diff
@@ -537,19 +537,7 @@ class EventImportForm(Form):
         label=_('Import'),
         validators=[
             DataRequired(),
-            WhitelistedMimeType({
-                'application/excel',
-                'application/vnd.ms-excel',
-                (
-                    'application/'
-                    'vnd.openxmlformats-officedocument.spreadsheetml.sheet'
-                ),
-                'application/vnd.ms-office',
-                'application/octet-stream',
-                'application/zip',
-                'text/csv',
-                'text/plain',
-            }),
+            WhitelistedMimeType(),
```

Comment:
This is not a file we store and could be downloaded by unsuspecting users after the fact, so the whitelist being strict isn't that important. That being said, we could probably trim it a little bit, since all we seem to accept for event imports are .xls and .xlsx files, it might be worth adding `application/x-ole-storage` though for old Excel files and `application/octet-stream` is probably fine here as well.

So I would keep the original whitelist, get rid of the bottom three and add `application/x-ole-storage`.